### DISCUSSION TITLE
SD Card Logging shows 0 bytes

### DISCUSSION THREAD

**Discussion by jc003** [cyberman54/ESP32-Paxcounter#968] [Q&A] [answered]
Hey guys, very new at this. I bought a LILYGO LoRa32 915Mhz ESP32 T3_V1.6.1 [off Amazon](https://www.amazon.com/dp/B09SHRWVNB?ref=ppx_yo2ov_dt_b_product_details&th=1) and followed documentation to configure an SD card to log data. What I see is a csv file named "paxcounter_4196089b.csv" but nothing is written after testing multiple times for 1-3 minutes. The card is formatted FAT32. The ttgov21new.h file has the following:

```
// enable only if you want to store a local paxcount table on the device
#define HAS_SDCARD  2      // this board has a SD MMC card-reader/writer
#define SDCARD_SLOTWIDTH 4 // 4-line interface

// Pins for SPI interface
#define SDCARD_CS   (13) // fill in the correct numbers for your board
#define SDCARD_MOSI (15)
#define SDCARD_MISO (2)
#define SDCARD_SCLK (14)
```

I checked the CS, MOSI, MISO, and SCLK numbers match up. The device otherwise displays count information on the OLED screen normally.

Any ideas where I might be missing something? Thanks!

**Comment by cyberman54** [comment]
Please check serial log output, if you see this line repeatingly:
_14:24:35.440 > I (70523) src/sdcard.cpp: writing data to SD-card_
If not, make sure you are using the current code of this repository. Current release is v3.6.0.

**Comment by jc003** [comment, marked as answer]
Ah it turns out I had to wait a little longer - about 6 minutes for the records to appear. Not sure why but good enough for my purpose.

**Comment by cyberman54** [comment]
There is a flushed write to minimize flash writes, that's why you see a delay. This is intentionally.
